nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -0,0 +1,99 @@
+# This module contains the basic configuration for building a graphical NixOS
+# installation CD.
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  imports = [ ./installation-cd-base.nix ];
+
+  users.extraUsers.nix-live = {
+    isNormalUser = true;
+    uid = 1000;
+    extraGroups = [ "wheel" "networkmanager" "video" ];
+    # Allow the graphical user to login without password
+    initialHashedPassword = "";
+  };
+
+  # Allow passwordless sudo from nix-live user
+  security.sudo = {
+    enable = lib.mkForce true;
+    wheelNeedsPassword = lib.mkForce false;
+  };
+
+  # Whitelist wheel users to do anything
+  # This is useful for things like pkexec
+  security.polkit.extraConfig = ''
+    polkit.addRule(function(action, subject) {
+      if (subject.isInGroup("wheel")) {
+        return polkit.Result.YES;
+      }
+    });
+  '';
+
+  services.xserver = {
+    enable = true;
+
+    # Don't start the X server by default.
+    autorun = mkForce false;
```

Comment:
Drop this I think. This is a graphical media, I would expect it to just start.

Reply:
I think this belongs in another PR.

I think we should add another boot entry so you can still choose not to start the graphical environment, at the same time we make the one that autostarts the graphical environment the default.

Reply:
Why would that be needed? I don't see the graphical env ever being not functional, and you can always just start it and switch to a tty if needed. If someone wanted to not have a graphical env they would use the other iso...